cpp/unreal_plugins/SpCore/Source/SpCore/Unreal.cpp

```diff
@@ -360,6 +361,7 @@ void Unreal::setPropertyValueFromString(const Unreal::PropertyDesc& property_des
         property_desc.property_->IsA(FFloatProperty::StaticClass())  ||
         property_desc.property_->IsA(FDoubleProperty::StaticClass()) ||
         property_desc.property_->IsA(FStrProperty::StaticClass())    ||
+        property_desc.property_->IsA(FNameProperty::StaticClass())   ||
```

Comment:
- consistently placed in the same order as in the previous usage, again a thoughtful choice 👌